controllers/user.controller.js

```diff
@@ -0,0 +1,45 @@
+const mongoose = require("mongoose");
+const createError = require("http-errors");
+
+const User = require("../models/User");
+const ERROR_MESSAGE = require("../constants");
+
+const getUserMaps = async (req, res, next) => {
+  const { id } = req.params;
+
+  try {
+    if (mongoose.isValidObjectId(id)) {
+      const error = createError(400, ERROR_MESSAGE.BAD_REQUEST);
+      throw error;
+    }
+
+    const userMaps = await User.findById(id).populate("myMaps").lean().exec();
+
+    const maps = userMaps.map(map => ({
+      id: map._id,
+      title: map.title,
+    }));
+
+    res.json({ maps });
+  } catch (err) {
```

Comment:
저는 
```suggestion
  } catch (error) {
```

이렇게 풀네임으로 사용하는데 다른분들은 어떻게 사용하시나요??

Reply:
풀네임으로 통일하는게 좋을 것 같아요!